tests/data/hetatm.pdb

```diff
@@ -0,0 +1,15 @@
+HEADER    THIS A DUMMY PDB FOR PDB-TOOLS TESTING                                
+TITLE     A RANDOM PDB                                                          
+COMPND    MOL_ID: 1;                                                            
+SOURCE    MOL_ID: 1;                                                            
+KEYWDS    PDB-TOOLS, TEST-PDB                                                   
+REMARK    ATOMS HAVE REPEATED NAMES WITHIN EACH RESIDUE.
+HETATM    1  C   UNK A   1      34.813  43.714  -0.371  1.00  0.00           C  
+HETATM    2  C   UNK A   1      35.081  45.036   1.305  1.00  0.00           C  
+HETATM    3  H   UNK A   1      34.200  45.604   1.876  1.00  0.00           H  
+HETATM    4  C   UNK A   1      35.381  43.810   2.140  1.00  0.00           C  
+HETATM    5 CA   UNK A   2      34.382  43.229   2.456  1.00  0.00          CA  
```

Comment:
Sorry, for my previous comments on the `CA`,
Please add also a test for carbon alpha `CA`, because some HETATM are also carbon alphas.
Test also against cases such as `HH1`.

Reply:
I added C/H/CA (carbon, hydrogen, calcium) because usually this use case is from ligand files that have only the element as the atom name. I'll add CA as an example that should be renamed to C1 (or whatever the number is).